tests/playwright/pages/editor-page.js

```diff
@@ -544,6 +559,24 @@ module.exports = class EditorPage extends BasePage {
 		return previewPage;
 	}
 
+	/*
+	 * @Description edit current page from the Front End.
+	 */
+	async editCurrentPage() {
+		const postId = await this.getPageIdFromFrontEnd();
+		await expect( postId, 'No Post/Page ID returned when calling getPageIdFromFrontEnd().' ).toBeTruthy();
+		await this.gotoPostId( postId );
+	}
+
+	async getPageId() {
+		const pageId = await this.page.evaluate( () => elementor.config.initial_document.id );
+		return pageId;
+	}
+
+	async getPageIdFromFrontEnd() {
+		const pageId = await this.page.evaluate( () => elementorFrontendConfig.post.id );
```

Comment:
@Barcov Should `pageId` not be returned?

Reply:
@louiswol94 Nice save, I've combined getPageIdFromFrontEnd() with getPageId() and included the return